### ISSUE TITLE
ui/README.md might be out of date

### ISSUE THREAD

**Issue by toolness** [18F/omb-eregs#637]
I was reading [`ui/README.md`](https://github.com/18F/omb-eregs/blob/master/ui/README.md) and noticed that it doesn't mention next.js, but it does mention `react-router` and `react-resolver`, neither of which are explicitly listed as dependencies in `package.json`.

My understanding is that this project once used a bespoke solution for next.js-esque functionality, but later migrated to next.js, so I'm wondering if this readme was written before that migration, and not updated to reflect it?

**Comment by cmc333333** [comment]
Agreed and you are accurate on timeline. This was written in April, long before the Next.js swap.
